.cirrus.yml

```diff
@@ -119,19 +123,25 @@ task:
   build_image_script: |
     DATE=$(date --utc +'%Y-%m-%dt%H-%M-%S')
 
+    # if TASK_NAME is not set use CIRRUS_TASK_NAME instead
+    if [[ ! -v TASK_NAME ]]; then
+      TASK_NAME=${CIRRUS_TASK_NAME}
+    fi
+    echo "TASK_NAME = ${TASK_NAME}"
```

Comment:
Why don't we just migrate everything over? It's a small enough file that I think "backward compatibility" hacks shouldn't be needed.